### PR TITLE
Fix/bootstrap turbo docs

### PR DESCRIPTION
**Pull request by Kerrialn**
Add a note and code example in the "Using Bootstrap CSS & JS" docs. 
If a user is using Turbo Drive with bootstrap to wrap their javascript in a `turbo:load` event listener.

### CODE REVIEW

File: frontend/encore/bootstrap.rst [hunk 1]
```diff
@@ -73,6 +73,23 @@ Now, require bootstrap from any of your JavaScript files:
         $('[data-toggle="popover"]').popover();
     });
 
+.. tip::
```
Comment (weaverryan) [inline frontend/encore/bootstrap.rst]: Maybe just a new short section to make it easy to find?

```
Using Bootstrap with Turbo
---------------------------
```

File: frontend/encore/bootstrap.rst [hunk 2]
```diff
@@ -73,6 +73,23 @@ Now, require bootstrap from any of your JavaScript files:
...
+
+    If you are using bootstrap with Turbo Drive, to allow your javascript to load on each page change, remember to wrap your javascript in a ``turbo:load`` event listener.
```
Comment (weaverryan) [inline frontend/encore/bootstrap.rst]: Add a line break here, after (approximately) the 75th character.

File: frontend/encore/bootstrap.rst [hunk 3]
```diff
@@ -73,6 +73,25 @@ Now, require bootstrap from any of your JavaScript files:
         $('[data-toggle="popover"]').popover();
     });
 
+Using Bootstrap with Turbo
+---------------------------
+
+If you are using bootstrap with Turbo Drive, to allow your javascript to load on each page change,
```
Comment (weaverryan) [inline frontend/encore/bootstrap.rst]:
```suggestion
If you are using bootstrap with Turbo Drive, to allow your JavaScript to load on each page change,
```

File: frontend/encore/bootstrap.rst [hunk 4]
```diff
@@ -73,6 +73,25 @@ Now, require bootstrap from any of your JavaScript files:
...
+remember to wrap your javascript in a ``turbo:load`` event listener.
```
Comment (weaverryan) [inline frontend/encore/bootstrap.rst]:
```suggestion
wrap the initialization in a ``turbo:load`` event listener:
```

**Review by weaverryan — COMMENTED** [review]
2 more tiny tweaks - sorry I missed these before!

**Review by weaverryan — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Kerrial, thanks for improving docs ... and congrats on your nice first Symfony Docs contribution 🎉

**Pull request merged**